Orange/widgets/evaluate/owpredictions.py

```diff
@@ -1094,11 +1260,40 @@ def rowCount(self, parent=QModelIndex()):
     def columnCount(self, parent=QModelIndex()):
         return 0 if parent.isValid() else self.__columnCount
 
+    def setRegErrorType(self, err_type):
+        self._reg_err_type = err_type
+
     def data(self, index, role=Qt.DisplayRole):
         row = self.mapToSourceRows(index.row())
         if role in (Qt.DisplayRole, Qt.EditRole):
             column = index.column()
-            return self._values[column][row], self._probs[column][row]
+            error_column = column % 2 == 1
+            column //= 2
+            if error_column:
+                if self._actual is None:
+                    return None
+                actual = self._actual[row]
+                if numpy.isnan(actual):
+                    return None
+                elif self._probs[column].size:
+                    return 1 - self._probs[column][row, int(actual)]
+                else:
+                    diff = self._values[column // 2][row] - actual
```

Comment:
`column` was already divided by 2 above (line 1271)!

I must say, there is quite a lot of `%2==1` and `//2` in this file... :)

Reply:
> column was already divided by 2 above (line 1271)!

It is disturbing that (seemingly thorough) tests didn't catch this because they used only a single predictor... :(

I fixed this and modified the tests to use two predictors.

> I must say, there is quite a lot of %2==1 and //2 in this file... :)

Apparently (at least) one too many. :)